client/src/hooks/useForgotPasswordForm.jsx

```diff
@@ -0,0 +1,103 @@
+import { useDisclosure, useToast } from '@chakra-ui/react';
+import { useRef, useState } from 'react';
+import useEmailValidation from 'hooks/useEmailValidation.jsx';
+import { userForgotPassword } from 'utils/api/users.js';
+import { sendEmail } from 'utils/api/system.js';
+
+const useForgotPasswordForm = () => {
+  const emailRef = useRef('');
+  const [emailIsValidated, setEmailIsValidated] = useState(true);
+  const [humanIsValidated, setHumanIsValidated] = useState(true);
+  const [buttonIsDisabled, setButtonIsDisabled] = useState(true);
+  const toast = useToast();
+  const [isLoading, setIsLoading] = useState(false);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { isAnEmail } = useEmailValidation();
+
+  const forgotPasswordHandleSubmit = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    const email = emailRef.current.value;
+
+    try {
+      const promise = await userForgotPassword({
+        email: email,
+      });
+      const url = window.location.href.substr(0, window.location.href.lastIndexOf('/'));
+
+      try {
+        await sendEmail({
+          to: [email],
+          subject: 'ERP Password Reset',
+          emailBody: `This is the ERP Team07,\n\nPlease reset your password using the following link: ${url}/reset/${promise.data}\n\nRegards.`,
```

Comment:
To avoid having to put /system/email public, you send the email from the server in the userForgotPassword call

Reply:
this way we ensure that only valid accounts can send emails